### PR TITLE
engine: Remove uses of Autohook from `runframe.cpp`

### PR DESCRIPTION
**Pull request by ASpoonPlaysGames**
### Code review:
- The original function is prefixed with `o_p` so any times where the old AUTOHOOK was calling the original function it should now be calling that instead.
- The hook function is prefixed with `h_` so any times where we would be wanting to call the hooked function from other functions should now be calling that instead

### Testing:
- Check logs to make sure that all of the changed hooks are being created properly
- ...that's all, honestly not sure why we hook this function anymore, probably a legacy hook or something at this point.

### CODE REVIEW

**Review by F1F7Y — APPROVED** [review]
Changes look good

File: primedev/engine/runframe.cpp [hunk 1]
```diff
@@ -3,17 +3,14 @@
...
-
-// clang-format off
-AUTOHOOK(CEngine__Frame, engine.dll + 0x1C8650,
-void, __fastcall, (CEngine* self))
-// clang-format on
+static void(__fastcall* o_pCEngine__Frame)(CEngine* self) = nullptr;
+static void __fastcall h_CEngine__Frame(CEngine* self)
 {
-	CEngine__Frame(self);
+	o_pCEngine__Frame(self);
```
Comment (Alystrasz) [inline primedev/engine/runframe.cpp:9]: Does this appear in the logs?
The old hook did (`Enabling hook CEngine__Frame`), but I can't find any trace of the new one.

Comment (GeckoEidechse) [reply primedev/engine/runframe.cpp:9]: @ASpoonPlaysGames

Comment (ASpoonPlaysGames) [reply primedev/engine/runframe.cpp:9]: non-auto hooks dont log when they are enabled iirc

**Comment by GeckoEidechse** [comment]
Testing in prod.

**Pull request merged**